resolver/addressable_resolver.go

```diff
@@ -68,17 +70,19 @@ func (r *URIResolver) URIFromDestination(dest apisv1alpha1.Destination, parent i
 			return "", err
 		}
 		return extendPath(url, dest.Path).String(), nil
-	} else if dest.URI != nil {
+	}
+
+	if dest.URI != nil {
```

Comment:
(no change needed)

technically `} else if {` is still fine, as it ends up being shorter, whereas `} else {` is longer because of the extra trailing `}`.

One notable exception to the `return ... } else { ... }` rule is when one of the leading `if` statements define variables, which would be out of scope when you drop the `} else {`, e.g.

```
if foo, err := bar(); err != nil {
  return err
} else {
  baz(foo)
}
```

is the same length as:
```
foo, err := bar()
if err != nil {
  return err
}
baz(foo)
```

Just FYI, in case you are interested in some of the underlying motivations behind the style madness 😅

Reply:
Good to know; I appreciate the feedback.